arch/arm64/boot/dts/freescale/crocodile.dts

```diff
@@ -0,0 +1,1090 @@
+// SPDX-License-Identifier: (GPL-2.0+ OR MIT)
+/*
+ * Copyright 2019 NXP
+ * Copyright (C) 2020 Leica Geosystems AG
+ */
+
+/dts-v1/;
+
+#include <dt-bindings/usb/pd.h>
+#include "imx8mm.dtsi"
+
+/ {
+	model = "FSL i.MX8MM EVK board";
```

Comment:
Please change the `model` binding to refer _Crocodile_ here. See example done for _DragonStaff_: https://github.com/hexagon-geo-surv/linux-leica/blob/75bb2dc432d020b98c15681f98f803a38c0ec00a/arch/arm64/boot/dts/freescale/dragonstaff.dts#L23